en/docs/develop/identity-governance-rest-api.md

```diff
@@ -3,16 +3,164 @@ template: templates/swagger.html
+    | Max Failed Attempts for ReCaptcha   | Recovery.Question.Password.ReCaptcha.MaxFailedAttempts   |  |
+    | Enable Username Recovery   | Recovery.Notification.Username.Enable   |  |
+    | Enable reCaptcha for Username Recovery   |  Recovery.ReCaptcha.Username.Enable  |  |
+    | Enable Internal Notification Management   | Recovery.Notification.InternallyManage  | Set false if the client application handles notification sending |
+    | Notify when Recovery Success   | Recovery.NotifySuccess   |  |
+    | Notify when Questions Based Recovery Starts   | Recovery.Question.Password.NotifyStart   |  |
+    | Recovery Link Expiry Time   | Recovery.ExpiryTime   |  |
+    | Enable forced challenge questions   | Recovery.Question.Password.Forced.Enable   | Force users to provide answers to challenge questions during sign in |
+    | Recovery callback URL regex   | Recovery.CallbackRegex   |  |
+    
+    * User Self Registration
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable Self User Registration  | SelfRegistration.Enable  | Enable self user registration |
+    | Enable Account Lock On Creation | SelfRegistration.LockOnCreation  | Lock user account during user registration |
+    | Internal Notification Management  | SelfRegistration.Notification.InternallyManage  | Set false if the client application handles notification sending |
+    | Enable reCaptcha  | SelfRegistration.ReCaptcha  | Enable captcha verification during self registration |
+    | User self registration verification link expiry time  | SelfRegistration.VerificationCode.ExpiryTime  | Set the number of minutes the user self registration verification mail would be valid |
+    | User self registration SMS OTP expiry time  | SelfRegistration.VerificationCode.SMSOTP.ExpiryTime  | Set the number of minutes that the SMS OTP would be valid |
+    | User self registration callback URL regex  | SelfRegistration.CallbackRegex  | User self registration callback URL regex |
+    | Manage Self-Sign-Up purposes  | _url_listPurposeSelfSignUp  | Click here to manage Self-Sign-Up purposes |
+    
+    * User Onboarding
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable User Email Verification  | EmailVerification.Enable  | Enable to trigger a verification notification during user creation |
+    | Enable Account Lock On Creation  | EmailVerification.LockOnCreation  | Lock user account during user creation |
+    | Internal Notification Management  | EmailVerification.Notification.InternallyManage  | Set false if the client application handles notification sending |
+    | Email verification code expiry time  | EmailVerification.ExpiryTime  | Set the number of minutes the email verification mail would be valid.(Negative value for infinite validity) |
```

Comment:
Set the number of minutes for which the email verification mail is valid. Setting it to a negative value results in infinite validity.